codeview/src/main/java/io/github/kbiakov/codeview/Utils.kt

```diff
@@ -10,6 +10,12 @@ import java.io.BufferedReader
 import java.io.InputStreamReader
 import java.util.concurrent.Executors
 
+class Utils {
```

Comment:
You can create only ```object``` with constant fields. At this moment it will contain only only one constant, so give name ```object Consts```.